DOC1898: [Partner Ops] Landing page (#685)
swan-io/docs: docs/topics/operations/index.mdx

---

> title: Supporting Your Operations Team

Reviewer: ```suggestion
title: Supporting your operations team
```

---

> # Supporting Your Operations Team

Reviewer: ```suggestion
# Supporting your operations team
```

## Overview {#overview}

These guides help your operations team manage day-to-day account and payment tasks on behalf of your users.

Each guide walks through a specific operational task, from verifying accounts to handling disputes.

:::note More guides are on the way

> This page will be updated as new guides are published.

Reviewer: ```suggestion
We'll update this page as new guides are published.
```

:::

## Account and user management {#account-user-management}

| Guide | Description |
|---|---|

> | [Manage account verification](/topics/operations/guide-account-verification/) | How to verify a user's account on their behalf. |

Reviewer: ```suggestion
| [Manage account verification](/topics/operations/guide-account-verification/) | How to manage account holder verification, from document collection to periodic reviews. |
```

| [Manage user identification](/topics/operations/guide-user-identification/) | How to manage identification requirements for your users. |

> | [Change an account admin or legal representative](/topics/operations/guide-change-account-admin-legal-representative/) | How to update the admin or legal representative on a user's account. |

Reviewer: ```suggestion
| [Change an account administrator or legal representative](/topics/operations/guide-change-account-admin-legal-representative/) | How to update the admin or legal representative on a user's account. |
```

> ## Payments operations {#payments-operations}

Reviewer: ```suggestion
## Payment operations {#payments-operations}
```

| Guide | Description |
|---|---|

> | [Manage mandates](/topics/operations/guide-manage-mandates/) | How to manage SEPA Direct Debit mandates for your users. |

Reviewer: ```suggestion
| [Manage SEPA Direct Debit mandates](/topics/operations/guide-manage-mandates/) | How to manage SEPA Direct Debit mandates for your users. |
```

## Disputes and chargebacks {#disputes-chargebacks}

| Guide | Description |
|---|---|
| [Card disputes and chargebacks](/topics/operations/guide-card-disputes-chargebacks/) | How to handle a card dispute and initiate a chargeback on behalf of your users. |

:::tip Can't find what you're looking for?

> Reach out to your Swan point of contact.

Reviewer: All five guides send readers to the [Partner support request form](https://partner-support.swan.io/hc/en-gb/requests/new) for escalation. Should this tip link the form instead of (or alongside) "your Swan point of contact"?

Author: This is more where they don't have the guide that they need - so they can reach out to their point of contact. Otherwise they'll raise a support ticket with "other"

> :::

Reviewer: ```suggestion
:::

```
Missing newline at end of file.

> :::

Reviewer: ```suggestion
:::

```
